packages/core/config/webpack.development.js

```diff
@@ -24,6 +24,39 @@ const getProxyConfig = () => {
         headers: {
           Connection: 'keep-alive',
         },
+        onError: function (err, req, res) {
+          if (err.code === 'ECONNRESET' || err.code === 'ECONNREFUSED') {
+            const msg = {
+              errors: [
+                {
+                  message:
+                    'The RedwoodJS API server is not available or is currently reloading. Please refresh.',
+                },
+              ],
+            }
+            // https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/203
+            // The HTTP 203 Non-Authoritative Information response status indicates that the request was successful
+            // but the enclosed payload has been modified by a transforming proxy from that of the origin server's 200 (OK) response
+            res.writeHead(203, {
+              'Content-Type': 'application/json',
+              'Cache-Control': 'no-cache',
+            })
+            res.write(JSON.stringify(msg))
+            res.end()
+          } else {
```

Comment:
I think this else block is problematic. If you through an AuthenticationError or an AuthorizationError for example - where the status code returned isn't 200, it will hide the actual message from the server.

I think we can remove this else block all together. What do you think?

Reply:
When I have

```
export const posts = () => {
  throw new AuthenticationError('Access denied')
  return db.post.findMany()
}
````

and `yarn rw dev web` then I get 

![image](https://user-images.githubusercontent.com/1051633/154078381-e0513fe6-933b-4d6c-9c92-84f6c130c91c.png)


which makes sense since the api is not running.

When running `yarn rw dev ` I get

![image](https://user-images.githubusercontent.com/1051633/154078488-112c6de2-55f7-4a1c-8d10-8673f6dcd9f8.png)


And this is keeps the "else" case:

```
 } else {
            const msg = {
              errors: [
                {
                  message:
                    'An error occurred. Please check your dev console for logs, or restart your RedwoodJS api server.',
                },
              ],
            }
            res.writeHead(203, {
              'Content-Type': 'application/json',
              'Cache-Control': 'no-cache',
            })
            res.write(JSON.stringify(msg))
            res.end()
          }

```

Can I get an example in the service? I'll try a few other errors